site/docs/master/namespace.md

```diff
@@ -1,23 +1,23 @@
-# Run in custom namespace
+# Run in a custom namespace
 
-You can run Velero in any namespace.
+The Velero installation and backups by default are run in the `velero` namespace. However, it is possible to customize it.
 
-First, ensure you've [downloaded & extracted the latest release][0].
+### 1) Customize the namespace on the cluster 
 
-Then, install Velero using the `--namespace` flag:
+During [the Velero installation][0], use the `--namespace` flag to inform Velero where to run.
 
 ```bash
 velero install --bucket <YOUR_BUCKET> --provider <YOUR_PROVIDER> --namespace <YOUR_NAMESPACE>
 ```
```

Comment:
My previous comment was about this command.

Reply:
If we don't want to repeat the whole command, then I suggest removing the command from this page and say 
> use the `--namespace` flag in  conjunction with the other flags in the `velero install` command as shown in the [install page][LinkToInstallPage]

WDYT?

Reply:
Yes, I like this. I think it's better. Thank you!